cilium-cli/connectivity/tests/errors.go

```diff
@@ -64,19 +64,21 @@ func NoErrorsInLogs(ciliumVersion semver.Version, checkLevels []string) check.Sc
 		failedCreategRPCClient, unableReallocateIngressIP, fqdnMaxIPPerHostname, failedGetMetricsAPI}
 	// The list is adopted from cilium/cilium/test/helper/utils.go
 	var errorMsgsWithExceptions = map[string][]logMatcher{
-		panicMessage:        nil,
-		deadLockHeader:      nil,
-		RunInitFailed:       nil,
-		emptyBPFInitArg:     nil,
-		RemovingMapMsg:      {stringMatcher("globals/cilium_policy")},
-		symbolSubstitution:  nil,
-		uninitializedRegen:  nil,
-		unstableStat:        nil,
-		missingIptablesWait: nil,
-		localIDRestoreFail:  nil,
-		routerIPMismatch:    nil,
-		emptyIPNodeIDAlloc:  nil,
-		"DATA RACE":         nil,
+		panicMessage:         nil,
+		deadLockHeader:       nil,
+		RunInitFailed:        nil,
+		emptyBPFInitArg:      nil,
+		RemovingMapMsg:       {stringMatcher("globals/cilium_policy")},
+		symbolSubstitution:   nil,
+		uninitializedRegen:   nil,
+		unstableStat:         nil,
+		missingIptablesWait:  nil,
+		localIDRestoreFail:   nil,
+		routerIPMismatch:     nil,
+		emptyIPNodeIDAlloc:   nil,
+		"DATA RACE":          nil,
+		envoyErrorMessage:    nil,
+		envoyCriticalMessage: nil,
```

Comment:
One note, we might want to start with 1.17 only first, similar to line below:
```
	if slices.Contains(checkLevels, defaults.LogLevelWarning) && ciliumVersion.GE(semver.MustParse("1.17.0")) {
		errorMsgsWithExceptions["level=warn"] = warningLogExceptions
	}
```

Reply:
I was just thinking from point of view of new CI failures that will potentially happen and slowly rolling out error coverage over stable branches. It's usually easier/faster to fix something on main branch rather than stable branches.